docs/tutorials/tokens/erc20.md

```diff
@@ -18,25 +18,25 @@ nav_order: 1
 ---
```

Comment:
In the "index" file in this directory, we call out a few well-known token standards, and for ERC-1155 we link to the "reference implementation".

I think we should remove that link and verbage within the ERC-1155 bullet, and instead add some notes at the bottom of that page to point out the "sample implementations" of ERC-1155, ERC-20, and ERC-721 contracts.

Reply:
Sorry, I'm unclean on exactly what you mean here. Remove the link. Got that. Then add notes that explain or link to what?

Reply:
Basically I think we should link to both connector implementations here:

<img width="829" alt="Use_tokens_-_Hyperledger_FireFly_Docs" src="https://user-images.githubusercontent.com/1993829/166306736-866d0ffc-2432-41a9-bb2c-68fd7d5dde07.png">

And I think it should be in the last paragraph (not in the bullet list).